pep-0723.rst

```diff
@@ -73,24 +73,39 @@ begun getting frustrated with the lack of unification regarding both tooling
 and specs. Adding yet another way to define metadata, even for a currently
 unsatisfied use case, would further fragment the community.
 
-A use case that this PEP wishes to support that other formats may preclude is
-a script that desires to transition to a directory-type project. A user may
-be rapidly prototyping locally or in a remote REPL environment and then decide
-to transition to a more formal project if their idea works out. This
-intermediate script stage would be very useful to have fully reproducible bug
-reports. By using the same metadata format, the user can simply copy and paste
-the metadata into a ``pyproject.toml`` file and continue working without having
-to learn a new format. More likely, even, is that tooling will eventually
-support this transformation with a single command.
+The following are some of the use cases that this PEP wishes to support:
+
+* A user facing CLI that is capable of executing scripts. If we take Hatch as
+  an example, the interface would be simply
+  ``hatch run /path/to/script.py [args]`` and Hatch will manage the
+  environment for that script. Such tools could be used as shebang lines on
+  non-Windows systems e.g. ``#!/usr/bin/env hatch run``. You would also be
+  able to enter a shell into that environment like other projects by doing
+  ``hatch -p /path/to/script.py shell`` since the project flag would learn
+  that project metadata could be read from a single file.
+* A script that desires to transition to a directory-type project. A user may
+  be rapidly prototyping locally or in a remote REPL environment and then
+  decide to transition to a more formal project layout if their idea works
+  out. This intermediate script stage would be very useful to have fully
+  reproducible bug reports. By using the same metadata format, the user can
+  simply copy and paste the metadata into a ``pyproject.toml`` file and
+  continue working without having to learn a new format. More likely, even, is
+  that tooling will eventually support this transformation with a single
+  command.
+* Tooling that wishes to modify metadata on behalf of the user, for example,
+  package managers or dependency update automation in CI.
```

Comment:
Is this a realisitic use-case? (i.e.: if you have a quick-and-dirty local script to achieve a task, would you be using CI (dependabot) or package management tools?).

Moreover, I think the *Rationale* should be focussed on the end user, not developers of tools. What benefit does the user derive from X scenario?

Reply:
Yes this is quite valid because many scripts are not quick-and-dirty but rather persist in version control as I mentioned below when talking about SREs. I will add a footnote.